content/posts/gitops/index.md

```diff
@@ -0,0 +1,497 @@
+---
```

Comment:
Please check the texts with https://languagetool.org. There are some missing commas and other small issues.

Reply:
Done.